### DISCUSSION TITLE
How to make animation loop infinite with Prismane <3 ?

### DISCUSSION THREAD

**Discussion by LuuKhoaHoc** [prismaneui/prismane#91] [Q&A] [answered]
I'am trying use hover Box and that loop bouncing animation

**Comment by spleafy** [comment]
Hi! Could you please tell me more about this bouncing animation, because Prismane's Animation component does not have a bouncing animation?

**Reply by LuuKhoaHoc** [reply]
I saw animation props in the Animation component had bounce animation. But I don't know how to make it repeat infinity

**Reply by spleafy** [reply, marked as answer]
The Animation component transitions between CSS properties based on a boolean flag. It does not use CSS animations, which means that the only way to animate infinitely is to toggle the boolean everytime the animation cycle ends, which will be a big hit to performance. 

Ultimately, the Animation could support vanilla CSS animations and I will add this as a feature in the next minor version.

**Reply by LuuKhoaHoc** [reply]
Yey, thank bro
